.github/resources/argo-lite/patch/argo_server_deployment.yaml

```diff
@@ -14,4 +14,4 @@ spec:
               memory: 500Mi
             requests:
               cpu: 20m
-              memory: 100m
+              memory: 100Mi
```

Comment:
seems unrelated -- extract to a separate PR?

Reply:
it's _kinda_ related, because we updated argo's deployment in the ci, and this unit was reporting errors

it should definitely be a new commit, so I added it as such